migration/sql-files/048-categories.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	name text
+);
+
+ALTER TABLE work_item_types ADD COLUMN categories_id uuid;
```

Comment:
@hector i did a few changes after this comment. The work_item_types table no more contains a "categories_id" column. There is a new table called "workitemtypes_categories" table which describes the relationship between workitemtypes and categories (basically which workitemtypes belong to which categories). It would be great if you could review the new implementation and let me know if some constraint is missing.